### PR TITLE
In progress

### PR DESCRIPTION
**Pull request by Emily774**
I have only done the html part of the webpage for now, will continue working on the CSS, but I could do with someone to look over what I've done so far. Great, thanks!

### CODE REVIEW

File: Portfolio1.html [hunk 1]
```diff
@@ -9,7 +9,8 @@
 
 <body>
 <!--Linked Logo and Internal Link -->
-
+<a target="_blank" href="https://www.history.com/topics/ancient-history/tutankhamen"> <img src="pexels-pixabay-33571.jpg" alt="Gold plated crest from the sarcophagi of Tutankharmun"></a>
```
Comment (damienscarpellini) [inline Portfolio1.html]: tag to go on a separate line. the closing </a> tag to go on the line after

File: Portfolio1.html [hunk 2]
```diff
@@ -9,17 +9,34 @@
 
 <body>
 <!--Linked Logo and Internal Link -->
-<a target="_blank" href="https://www.history.com/topics/ancient-history/tutankhamen"> <img src="pexels-pixabay-33571.jpg" alt="Gold plated crest from the sarcophagi of Tutankharmun"></a>
-<a href="#main-text">Skip to Main Text</a>
+<a target="_blank" href="https://www.history.com/topics/ancient-history/tutankhamen"> <img src="pexels-pixabay-33571.jpg" alt="Gold plated crest from the sarcophagi of Tutankharmun" width="100"></a>
```
Comment (damienscarpellini) [inline Portfolio1.html]: tag on next line down, closing </a> tag on line after

File: Portfolio1.html [hunk 3]
```diff
@@ -9,17 +9,34 @@
...
+	<button>STAFF</button>
+	<button>CONTACT</button>
+</div>
 
 <!--Central Image and Heading1 -->
+<img src="pexels-pixabay-260024.jpg" alt="A lone chesspiece stands between the two sides wearing a golden crown" width="">
+<h1>The First of Many Titles</h1>
 
 <!--Second heading and Text-box 1 + 2xlinks -->
+<h1>The second of Many Titles</h1>
```
Comment (damienscarpellini) [inline Portfolio1.html]: second use of h1 should only be one h1 on page

File: Portfolio1.html [hunk 4]
```diff
@@ -9,17 +9,34 @@
...
+<div>
+	The Portuguese man o' war (Physalia physalis), also known as the man-of-war, bluebottle, or floating terror is a <a href="https://en.wikipedia.org/wiki/Hydrozoa">marine hydrozoan</a> found in the <a href="https://en.wikipedia.org/wiki/Atlantic_Ocean">Atlantic Ocean</a> and <a href="https://en.wikipedia.org/wiki/Indian_Ocean">Indian Oceans</a>. It is considered to be the same species as the Pacific man o' war, which is found mainly in the Pacific Ocean.
```
Comment (damienscarpellini) [inline Portfolio1.html]: enclose in a p tag

File: Portfolio1.html [hunk 5]
```diff
@@ -38,20 +38,30 @@ <h1>The Second of Many Titles</h1>
...
 				<p> Dialogues between a Doctor of Divinity and a Student in the Laws of England is a legal treatise by Christopher St. Germain, first published in the early 16th century. As its name suggests, the work is structured as a set of dialogues between the eponymous doctor, a doctor of divinity; and a student of the English common law.</p>
 	
 		<h3>Sub-heading 2</h3>
-			<h4>Date</h4>
+			<h4>19th of January 2021</h4>
 				<p> Doctor and Student explores the relationship between the common law and equity and distinguishes a number of sources of legal principles. It was an important text for English law students at least until William Blackstone's Commentaries on the Laws of England was published in the mid-18th century.</p>
 </div>
 <!--Images and titles -->
+<div>
+	<label for="">ImageTitle1</label>
```
Comment (damienscarpellini) [inline Portfolio1.html]: no id

File: Portfolio1.html [hunk 6]
```diff
@@ -38,20 +38,30 @@ <h1>The Second of Many Titles</h1>
...
+	<img src="pexels-csongor-kemény-3933541.jpg" alt="Image of an iron statue of a regal looking king with a halo behind his head situated amongst the trees in cloud" width="50" id="">
```
Comment (damienscarpellini) [inline Portfolio1.html]: no id

File: Portfolio1.html [hunk 7]
```diff
@@ -9,11 +9,33 @@
...
+	<!--Skip to button -->
 <div class="skip-to">
 	<a href="#main-text">Skip to Main Text</a>
 </div>
+
+	<!--Sign in -->
+<form>
+		<!--Email -->
+	<label for="email">Email</label>
+<input type="email" name="email" id="email"/>
```
Comment (damienscarpellini) [inline Portfolio1.html]: indent inline with the above tag

File: Portfolio1.html [hunk 8]
```diff
@@ -9,11 +9,33 @@
...
+
+		<!--Password -->
+	<label for="password">Password</label>
+<input type="password" name="password" id="password"/>
```
Comment (damienscarpellini) [inline Portfolio1.html]: indent inline with the above tag

File: Portfolio1.html [hunk 9]
```diff
@@ -9,11 +9,33 @@
...
+		<!--Checkbox -->
+	<label for="remember">Remember Me</label>
+<input type=“checkbox” name=“remember” id="remember"/>
```
Comment (damienscarpellini) [inline Portfolio1.html]: indent inline with the above tag

File: Portfolio1.html [hunk 10]
```diff
@@ -9,11 +9,33 @@
...
+
+		<!--Submit button -->
+<input type="submit" value="submit"/>
```
Comment (damienscarpellini) [inline Portfolio1.html]: indent inline with the above tag

File: Portfolio1.html (hunk 5 above)
Comment (damienscarpellini) [reply Portfolio1.html]: i think you change this later on

File: Portfolio1.html (hunk 6 above)
Comment (damienscarpellini) [reply Portfolio1.html]: i think you change this later on

**Review by damienscarpellini — CHANGES_REQUESTED** [review]
Good stuff. A few minor things i've picked up on but nothing that seems to break the code. also, need to change the commit to the develop branch not the main branch.

File: Portfolio1.html [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+	<link rel="stylesheet" type="text/css" href="portfolio1.css">
+</head>
+
+<body>
+<!--Linked Logo and Internal Link -->
+
+	
+<div class="top-bar">
+	<!--Sarcophagi logo -->
+	<a target="_blank" 	href="https://www.history.com/topics/ancient-history/tutankhamen"> <img src="pexels-pixabay-33571.jpg" alt="Gold plated 	crest from the sarcophagi of Tutankharmun, with a link to the history of the pharoh" width="100"></a>
```
Comment (karenschulz) [inline Portfolio1.html]: line 16 alt="Gold plated 
don't have a closing "

Comment (karenschulz) [reply Portfolio1.html]: Loved you had comments, easier to read

**Comment by Emily774** [comment]
This pr should have been put into the develop branch, so I've shifted it over. New pull request: https://github.com/Emily774/Portfolio-Webpage/pull/2#issue-558336889

**Pull request closed without merging**